### ISSUE TITLE
Could you release a new version?

### ISSUE THREAD

**Issue by tricknotes** [lassebunk/gretel#76]
I found some unreleased feature in master. (diff: https://github.com/lassebunk/gretel/compare/v3.0.8...025419b5 )

I can use it via bundler.
However It is useful for us if we can just install via `gem install`.

**Comment by aldefouw** [comment]
I agree wholeheartedly.

I've just spent a few hours trying to figure out why I was getting deprecation warnings on Rails 5.  I saw that the fix had been merged in a long time ago.  I never looked at whether it was actually released though.  

Just bundle updated to the newest release.  

After some further investigation, I came to realize that the merged fix for Rails 5 deprecation warnings on this gem were never actually released in a new version and are just sitting on master.

The last release - 3.0.8 - was made in 2014.

I think we'd all really appreciate a new release.
